arcgis-ios-sdk-samples/Shared resources/Military messages/Mil2525DMessages.xml

```diff
@@ -6,6 +6,7 @@
         <_id>102991b1-ad06-412a-9e46-1d6464a4ba64</_id>
         <_control_points>-226750.39376,6680971.38044;-226585.9422,6680967.75384;-226425.36901,6680941.12366;-226273.55313,6680892.29906;-226135.1074,6680822.76355;-226014.23842,6680734.62992;-225914.61875,6680630.57608;-225839.27527,6680513.76364;-225790.49725,6680387.74189;-225769.7668,6680256.33995;-225777.71379,6680123.55039;-225814.09677,6679993.40795;-225877.81024,6679869.86695;-225966.91832,6679756.68112;-226078.71349,6679657.28955;-226209.79892,6679574.7122;-226356.19165,6679511.45815;-226513.4436,6679469.44934;-226676.77675,6679449.96219;-226841.22831,6679453.58879;-227001.8015,6679480.21897;-227153.61738,6679529.04357;-227292.06311,6679598.57908;-227412.93209,6679686.71271;-227512.55176,6679790.76655;-227587.89524,6679907.57899;-227636.67326,6680033.60074;-227657.40371,6680165.00268;-227649.45672,6680297.79224;-227613.07374,6680427.93468;-227549.36027,6680551.47568;-227460.25219,6680664.66151;-227348.45701,6680764.05308;-227217.37158,6680846.63043;-227070.97886,6680909.88448;-226750.39376,6680971.38044</_control_points>
         <_wkid>3857</_wkid>
+        <sic>GFGPOAO-------X</sic>

```

Comment:
It seems a bit odd that this file is being updated. All the other changes are specific to the Graphics Overlay (Dictionary Renderer) sample, but that sample no longer uses this file. Including these changes in this PR means that two samples have to be tested instead of one.

Reply:
The sample still uses this file, see [line 63](https://github.com/Esri/arcgis-runtime-samples-ios/pull/1128/files#diff-f137eb0a526906d151086880c0c080aa8282bfcd21380095b7478700f097e158R63). Basically to render the symbols, we need... 

- the stylx file, which contains the symbols
- the xml file, which contains the geometry, such as control points for polylines, single points to place markers, etc.

These reverted changes in the xml file are requested by symbology team, in order to support the `Ordered Anchor Points` attribute in the style.

Reply:
Oh! This isn't the style file. Right. Never mind!